xendit-java-lib/src/main/java/com/xendit/network/BaseRequest.java

```diff
@@ -24,7 +24,7 @@
   public <T> T request(
       RequestResource.Method method, String url, Map<String, Object> params, Class<T> clazz)
       throws XenditException {
-    return staticRequest(method, url, new HashMap<>(), params, clazz);
+    return request(method, url, new HashMap<>(), params, clazz);
```

Comment:
What `request` method is this? @kahirul

Reply:
Is it calling itself?

Reply:
Nope, invoking overloaded version with `headers` as parameter